main.go

```diff
@@ -102,7 +102,7 @@ func NewGame(g *Game) {
 	g.Background = bg
 
 	// Create space for collision detection
-	g.Space = resolv.NewSpace(level.Width, level.Height, 16, 16)
+	g.Space = resolv.NewSpace(level.Width, level.Height, 8, 8)
```

Comment:
I don't think you should make the whole grid smaller because of the dog, 16x16 was effective for most of the tiles, with 8x8 all tiles and every entity except dog will cover 4 tiles at a time at least.

Reply:
Yes, you are right, I will change it back